marklogic-data-hub-central/ui/src/components/entities/mapping/source-entity-map/source-to-entity-map.test.tsx

```diff
@@ -37,7 +38,225 @@ describe('RTL Source-to-entity map tests', () => {
         fireEvent.click(getByText('Clear'));
         expect(getByText('Clear')).toBeEnabled();
         expect(getByText("concat(name,'-NEW')")).toBeInTheDocument();
-        console.log(container);
+    });
+
+    test('Filtering Name column in Source and Entity tables',() => {
```

Comment:
need to repeat this test using proper xml doc and convert this one to use json doc.

Reply:
This will be taken care of in a separate story.